toolkits/global/packages/global-banner/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@springernature/global-banner",
-  "version": "0.1.1",
+  "version": "0.2.0",
   "license": "MIT",
-  "description": "Represents a box with text",
+  "description": "Full width banner used for displaying informational messages",
   "keywords": [],
   "author": "Springer Nature",
   "peerDependencies": {
```

Comment:
not related to our changes but could bump this to 10.0.0 😁

Reply:
^^ global-context 😄

Reply:
🤦‍♂that's what i meant